akka-cluster/src/main/scala/akka/cluster/Gossip.scala

```diff
@@ -311,13 +311,13 @@ private[cluster] final case class Gossip(
   }
 
   def pruneTombstones(removeEarlierThan: Gossip.Timestamp): Gossip = {
-    val newTombstones = tombstones.filter { t ⇒ t._2 < removeEarlierThan }
+    val newTombstones = tombstones.filter { case (_, timestamp) ⇒ timestamp < removeEarlierThan }
```

Comment:
I think there were a comment about allocations. I think the pattern matcher is smart enough to not allocate anything for this.